test/services/fileupload/FileServiceSpec.scala

```diff
@@ -188,7 +188,7 @@ class FileServiceSpec extends AnyFreeSpec with SpecBase with BeforeAndAfterEach
         val updatedFile = uploadedFile.copy(downloadUrl = "drafts/DRAFT000000000/foobar")
 
         val expectedAnswers = userAnswers
-          .set(UploadSupportingDocumentPage, updatedFile)
+          .set(UploadLetterOfAuthorityPage, updatedFile)
```

Comment:
would it be worth keeping and adding this as separate test, for different branch?

Reply:
it has the other branch in other tests no?

Reply:
Is ok. Just thought it may make it clearer at quick glance, as 2 differences in test.